src/integration-tests/tcp/freedom-module.ts

```diff
@@ -59,12 +59,12 @@ parentModule.on('shutdown', () => {
     var client = new tcp.Connection({endpoint: endpoint});
     server.connectionsQueue.setSyncHandler((connection:tcp.Connection) => {
       client.onceConnected.then(() => {
-        server.shutdown();
+    server.shutdown();
```

Comment:
Sorry, I had my diff tool ignoring leading whitespace for some strange reason...